packages/dropdown/src/useDropdown.ts

```diff
@@ -27,17 +30,22 @@ export const useDropdownDomEvent = (dropdownChildren, triggerElm, _instance) =>
     })
   }
 
+  function resetTabindex(ele) {
+    removeTabindex()
+    ele?.setAttribute('tabindex', '0')
+  }
+
   function handleTriggerKeyDown(ev: KeyboardEvent) {
     const keyCode = ev.keyCode
-    if ([38, 40].includes(keyCode)) { // up/down
+    if ([eventKeys.up, eventKeys.down].includes(keyCode)) { // up/down
       removeTabindex()
-      _instance.resetTabindex(menuItems.value[0])
-      menuItems.value[0].focus()
+      resetTabindex(menuItems.value[0])
+      ;(menuItems.value[0] as HTMLButtonElement).focus()
       ev.preventDefault()
       ev.stopPropagation()
-    } else if (keyCode === 13) { // space enter
+    } else if (keyCode === eventKeys.enter) { // space enter
       _instance.handleClick()
-    } else if ([9, 27].includes(keyCode)) { // tab || esc
+    } else if ([eventKeys.tab, eventKeys.esc].includes(keyCode)) { // tab || esc
```

Comment:
When you added explicit key code here I do think the comments can be removed now